src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.tsx

```diff
@@ -47,8 +47,10 @@ function CurrentOrgPlan() {
 
   return (
     <div className="w-full lg:w-4/5">
+      {/* subscriptionDetail populated means possible cancel at period end cancellation while
+      planUpdatedNotification.isCancellation signifies an auto-refund cancellation */}
       {accountDetails?.subscriptionDetail ||
-      planUpdatedNotification.isRefundedCancellation ? (
+      planUpdatedNotification.isCancellation ? (
```

Comment:
Im wondering if cancellation or auto refund is the more important aspect of this variable name

Reply:
The thought comes from the variable "isCancellation" only being true for a very small percentage of total cancellations

Reply:
isCancellation is always true when we want to show this component regardless of `accountDetails?.subscriptionDetail`'s value so will be removing that check